typescript/src/languages/en-US/commands/management.json

```
@@ -357,21 +359,58 @@
 		"examples": [
 			"#member-logs",
 			"here"
+		],
+		"reminder": "All messages are in embeds so you will need to enable the permission **{{EMBED_LINKS, permissions}}** for Skyra."
+	},
+	"setMemberRemoveLogsDescription": "Set the member remove logs channel.",
+	"setMemberRemoveLogsExtended": {
+		"usages": [
+			"here/TextChannel"
+		],
+		"extendedHelp": "This command helps you setting up the member remove log channel, which will send messages each time a user leaves, is kicked, or banned.\nIn the case of the last two, it will send a special \"User Kicked\" or \"User Banned\" event.",
+		"explainedUsage": [
+			[
+				"channel",
+				"A TextChannel. You can either put the name of the channel, tag it, or type in \"here\" to select the channel the message was sent."
```

```suggestion
				"A TextChannel. You can either put the name of the channel, tag it, or type in \"here\" to configure the channel this command was used in."
```

Also make the same change above for `setMemberAddLogsExtended`. I cannot use suggestion because GitHub says it's unchanged code.